packages/flutter_test/lib/src/_binding_io.dart

```diff
@@ -15,8 +15,7 @@ import 'binding.dart';
 
 /// Ensure the appropriate test binding is initialized.
 TestWidgetsFlutterBinding ensureInitialized([@visibleForTesting Map<String, String>? environment]) {
-  environment ??= Platform.environment;
-  if (environment.containsKey('FLUTTER_TEST') && environment['FLUTTER_TEST'] != 'false') {
+  if (environment ?? Platform.environment case {'FLUTTER_TEST': != 'false'}) {
```

Comment:
Could this just be

```suggestion
  environment ??= Platform.environment;
  if (environment['FLUTTER_TEST'] != 'false') {
```

?

Reply:
Probably not, since if a `FLUTTER_TEST` environment variable isn't provided, then it'd evaluate to `if (null != 'false')`.

I'll probably just revert here to make things easy 👍